pkg/convert/convert.go

```diff
@@ -29,8 +33,20 @@ func escapeSpecialChars(message string) string {
 	return message
 }
 
-// removeEnclosingBrackets replaces '{' and '}', temporarily maintain only the singular form.
-func removeEnclosingBrackets(message string) string {
-	replacer := strings.NewReplacer("{", "", "}", "")
-	return replacer.Replace(message)
+func getMsg(message string) (string, error) {
+	nodes, err := messageformat.Parse(message)
+	if err != nil {
+		return "", fmt.Errorf("parse message: %w", err)
+	}
+
+	for _, node := range nodes {
+		nodeTxt, ok := node.(messageformat.NodeText)
+		if !ok {
+			return "", errors.New("convert node to messageformat.NodeText")
+		} else {
+			return nodeTxt.Text, nil
+		}
```

Comment:
if you have `if... else`, do not negate the expression, it reduces readability. Do not use the `else` block if `if` returns.

```suggestion
		if ok {
			
			return nodeTxt.Text, nil
		}
		
		return "", errors.New("convert node to messageformat.NodeText")
```